docs/windows.md

```diff
@@ -0,0 +1,195 @@
+# Antrea on Windows
+
+## Overview
+Antrea supports to running on Windows worker Node, and setups an overlay network
```

Comment:
"supports to run on". I guess maybe just "Antrea supports Windows Node".

Reply:
"sets up"

Reply:
Done.